spec/spec_helper.rb

```diff
@@ -112,7 +112,10 @@ def only(*whitelist)
   end unless method_defined?(:only)
 end
 
-# set default time zone in TZ environment variable
-# which will be used to set session time zone
-ENV['TZ'] ||= 'Europe/Riga'
-# ENV['TZ'] ||= 'UTC'
+# Set Oracle Database session timezone
+# this setting needs to be in place to make sure that Oracle properly represents
+# the timestamp format on your local machine
+# Do not use 'OS_TZ' value or fixed offset values like '05:00', for daylight saving-enabled timezones
+# The setting cannot be derived directly from Windows operating system or ruby, as on the Windows machines,
+# timezone names are non-standard ones and are subject to translations to different languages.
+ENV['ORA_SDTZ'] ||= 'Europe/London'
```

Comment:
Yes, Oracle time zones names are Oracle-specific - that's not a Windows-specific issue I think. I agree about the approach with setting ORA_SDTZ.

Just I propose something like following in order to avoid the hassle of having local custom settings in version-controlled files:
* Add `spec/support/custom_config.rb` to `.gitignore`
* If anyone wants to set specific ORA_SDTZ: this should be done in spec/support/custom_config.rb
* We may provide commented sample `spec/support/custom_config.rb.sample` or/and maybe some defaults e.g. in `spec_helper`.